standalone-examples/src/main/java/io/pravega/example/binary/NullReader.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.binary;
```

Comment:
Why is this reader in `binary`  package?

Reply:
I will rename this `noop` to go with the sample name.